Packages/com.llealloo.audiolink/Runtime/Scripts/ytdlpPlayer.cs

```diff
@@ -59,11 +60,27 @@ public enum Resolution
 
         void OnEnable()
         {
+            if (audioLink == null)
+            {
+                audioLink = GetComponentInParent<AudioLink>();
+                if (audioLink == null)
+                {
+                    audioLink = FindObjectOfType<AudioLink>();
```

Comment:
It looks like this is deprecated since 22.3 (https://docs.unity3d.com/6000.1/Documentation/ScriptReference/Object.FindObjectOfType.html) and one should use Object.FindFirstObjectByType ?

Reply:
Oh haven't realized they have something new. Yes it can be replaced with `FindFirstObjectByType`. Should I change it or you will change it while merging?

Reply:
I can do it